### DISCUSSION TITLE
Vocês usam Windows ou Linux para desenvolvimento no dia a dia?

### DISCUSSION THREAD

**Discussion by huogerac** [devpro-br/hackernews-clone#1] [Q&A]
Existe uma diversidade gigante na escolha de S.O, IDE etc.. Embora isto não seja problema algum, e digo mais, cada vez isto está menos relevante! É importante saber a preferência do pessoal que vai participar deste repositório!

**Vocês estão utilizando Windows ou Linux para desenvolver software?**

Veja também:
Como preparar ambiente frontend #5

**Comment by felsb3** [comment]
Linux

Em seg., 27 de set. de 2021 às 15:10, Roger Camargo <
***@***.***> escreveu:

**Comment by huogerac** [comment]
Já tem um bom tempo que uso Linux (Ubuntu, Mint ou elementary OS)

**Comment by Gabrieltggv** [comment]
estou usando o Linux(ArchLinux).

**Comment by Brutos212** [comment]
Linux é Vida(Qual que Distro)

**Comment by wfoschiera** [comment]
Linux tb

**Comment by Riverfount** [comment]
GNU/Linux Debian Sid forever

**Comment by luxu** [comment]
unico q usa Win 10....

**Reply by joaogarciadelima** [reply]
eu tbm

**Comment by serlus** [comment]
Uso linux. ubuntu com interface KDE atualmente, mas meu conhecimento é limitado ao ubuntu tendo experimentado apenas interfaces diferentes. (ainda tenho curiosidade trabalhar com Debian e distro baseadas no Arch)

**Comment by yzakius** [comment]
Linux.

**Comment by diegomaranhao** [comment]
Linux do Pc do trabalho, Windows com WSL2 no PC pessoal.

**Comment by victoraugusto6** [comment]
Linux, distro **Pop!_OS**.

**Reply by marcospsviana** [reply]
Linux Manjaro
